sorc/link_workflow.sh

```diff
@@ -162,15 +162,22 @@ cd "${HOMEgfs}/parm/ufs" || exit 1
 ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_model.fd/tests/parm/noahmptable.tbl" .
 
 cd "${HOMEgfs}/parm/post" || exit 1
```

Comment:
I think we should follow the UPP and put these into separate `gfs` and `gefs` subdirectories now.

Reply:
OK, fixed.